app/src/main/res/layout/faq_fragment.xml

```diff
@@ -6,24 +6,18 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <com.google.android.material.appbar.AppBarLayout
-      android:id="@+id/faq_app_bar_layout"
+    <LinearLayout
       android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:theme="@style/ThemeOverlay.AppCompat.ActionBar"
+      android:layout_height="match_parent"
+      android:orientation="vertical"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent">
 
-      <androidx.appcompat.widget.Toolbar
-        android:id="@+id/faq_toolbar"
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/faq_fragment_recycler_view"
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:background="@color/oppiaPrimaryDark"
-        android:textSize="20sp"
-        app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
-        app:title="@string/frequently_asked_questions"
-        app:titleTextColor="@color/white" />
-    </com.google.android.material.appbar.AppBarLayout>
+        android:layout_height="match_parent" />
+    </LinearLayout>
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
done